internal/examples/json-keyvalue/server/main.go

```diff
@@ -102,6 +104,12 @@ func do() error {
 			return err
 		}
 		inbound = tchannelTransport.NewInbound()
+	case "grpc":
+		listener, err := net.Listen("tcp", "127.0.0.1:24038")
```

Comment:
This is a recommendation that Prashant had previously. I don't actually think it makes a difference, but I'm trying to do it for new connections I make.